src/software/thunderscope/thunderscope.py

```diff
@@ -458,27 +489,53 @@ def configure_robot_diagnostics_layout(self, dock_area, proto_unix_io):
         log_dock = Dock("Logs")
         log_dock.addWidget(self.diagnostics_widgets["log_widget"])
 
+        if not load_fullsystem:
+            self.diagnostics_widgets["robot_view"] = self.setup_robot_view(
+                proto_unix_io, True
+            )
+            robot_view_dock = Dock("RobotView")
+            robot_view_dock.addWidget(self.diagnostics_widgets["robot_view"])
+            self.toggle_robot_connection_signal = self.diagnostics_widgets[
+                "robot_view"
+            ].toggle_robot_connection_signal
+
+        self.diagnostics_widgets[
+            "fullsystem_connect"
+        ] = self.setup_fullsystem_connect_widget(proto_unix_io)
+        fullsystem_connect_dock = Dock("Fullsystem")
```

Comment:
nit: We should update this to something more descriptive like "Input"

Reply:
just noticed that the comment seemed very ambiguous, I'm referring to the title of the dock here. Feel free to use a different name that you think seems fit.
```suggestion
        fullsystem_connect_dock = Dock("Input")
```